.github/workflows/validate-bicep.yml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+name: validate-bicep
+on: [pull_request, workflow_dispatch]
```

Comment:
can we scope this to execute on changes to just `.bicep` modules and/or the contents of `src/bicep`?

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths

Reply:
Of course!  Done!